src/engine/LoadingScreen.cs

```diff
@@ -25,17 +26,24 @@ public class LoadingScreen : Control
 
     private static LoadingScreen instance;
 
+    private readonly Random random = new Random();
+
     private Label artDescription;
     private Label loadingMessageLabel;
     private Label loadingDescriptionLabel;
     private Label tipLabel;
     private Control spinner;
 
-    private string loadingMessage = TranslationServer.Translate("LOADING");
+    private bool visible;
```

Comment:
This should be named "wasVisible" to not be confused regarding the Godot `Visible` property this class already has.